src/main/resources/swagger/api-docs.yaml

```diff
@@ -840,6 +840,39 @@ paths:
       tags:
         - Resources
 
+  /api/resources/v1/{resourceTypeName}/{resourceId}/authDomain:
+    get:
+      summary: List the groups in the Auth Domain for a resource
+      responses:
+        200:
+          description: List of groups that compose the Auth Domain for this resource. Empty if an Auth Domain has not been set
+          schema:
+            type: array
+            items:
+              type: string
+        403:
+          description: You do not have permission to perform this action on the resource
+        404:
+          description: Resource type or resource does not exist or you are not a member of any policy on the resource
+        500:
+          description: Internal Error
```

Comment:
I'd prefer `Internal Server Error`

Reply:
I can do that, but `Internal Error` is consistent with the other 500 status responses across the Swagger doc

Reply:
maybe we should update all of them? 🤔 (https://tools.ietf.org/html/rfc7231#section-6.6.1)